freezed_provider_value_notifier/lib/models.dart

```diff
@@ -0,0 +1,61 @@
+// Copyright 2018 The Flutter Architecture Sample Authors. All rights reserved.
+// Use of this source code is governed by the MIT license that can be found
+// in the LICENSE file.
+
+import 'package:todos_app_core/todos_app_core.dart';
+import 'package:todos_repository_core/todos_repository_core.dart';
+
+enum AppTab { todos, stats }
+
+enum ExtraAction { toggleAllComplete, clearCompleted }
+
+class Todo {
```

Comment:
Feeezed doesn't support default values & methods yet. When it will, I'll update this sample. But for now the syntax wouldn't be too cool.

Although this does bring an interesting point. I wonder if Freezed could automatically generate such `fromEntity`/`toEntity` 😄

Reply:
Gotcha! Interesting limitation for sure...

Reply:
Well, it's actually a Dart limitation 🤷‍♂ 
For some reasons, factory using `factory Class() = Redirect;` cannot use default values.

Methods can be added with extensions so it's fine